src/fixtures/translation_plugin_template/README.md

```diff
@@ -0,0 +1,23 @@
+A Kibana translation plugin structure. 
```

Comment:
We aim to amuse! :) I have made changes in the other repo as https://github.com/elastic/generator-kibana-plugin/pull/45